lib/src/git_backend.rs

```diff
@@ -1430,7 +1430,7 @@ Git or other tools that are unfamiliar with jj.
 
 The .jjconflict-* directories represent the different inputs to the conflict.
 For details, see
-https://martinvonz.github.io/jj/prerelease/git-compatibility/#format-mapping-details
+https://jj-vcs.github.io/jj/prerelease/git-compatibility/#format-mapping-details
```

Comment:
This file also contains a link to `https://github.com/martinvonz/jj` which should probably be updated.

Reply:
Oh, good point! There are tons of those links. I'll fix all of them, including the ones in the changelog (it feels a bit weird to update the changelog, but I think it makes sense in this case).

Reply:
Done.